homeassistant/components/template/entity.py

```diff
@@ -49,7 +52,8 @@ def __init__(
                 optimistic is None and assumed_optimistic
             )
 
-        if (object_id := config.get(CONF_OBJECT_ID)) is not None:
+        if (default_entity_id := config.get(CONF_DEFAULT_ENTITY_ID)) is not None:
+            _, object_id = default_entity_id.split(".")
```

Comment:
I think we prefer this pattern:
```suggestion
            _, _, object_id = default_entity_id.partition(".")
```